doc/GregorioNabcRef.tex

```diff
@@ -89,6 +89,246 @@
 }
 
 {
+\setlength\parindent{0cm}
+
+The \texttt{nabc} language allows to describe some adiastematic neumes, for
+now just the St. Gallen style.  The language is partially based on Dom
+Eugène Cardine's Table of neumatic signs, but for more complex neumes
+doesn't always match how the neumes are called; instead attempts to make it
+easier to compose complex neumes from basic glyphs.  To describe
+adiastematic neumes in \texttt{gabc}, the header should contain
+\texttt{nabc-lines: 1;} line, like:
+
+\begin{verbatim}
+nabc-lines: 1;
+(f3) AL(ef~|ta>)le(fg/hggf|peclhgpi)lú(ef~|ta>)ia.(f.|ta-) (,)
+(ii//|bv-|gh!ivHG//|vi-hhppu2su1sut1|fhg/|to|eef.|pt) (;)
+\end{verbatim}
+
+The \texttt{nabc} snippets are then separated by \texttt{|} character from
+\texttt{gabc} snippets.  Every \texttt{gabc} snippet may be followed by
+up to \texttt{nabc-lines} \texttt{nabc} snippets, followed optionally
+again by \texttt{gabc} and so on.  A single \texttt{nabc} snippet is not
+split into multiple lines, so for larger melismatic pieces it is desirable
+to synchronize the \texttt{gabc} snippets with corresponding \texttt{nabc}
+snippets.
+
+Each \texttt{nabc} snippet consists of a sequence of \textit{complex neume
+descriptors}.  Each \textit{complex neume descriptor} consists of
+optional \textit{horizontal spacing adjustment descriptor}, then \textit{complex glyph
+descriptor}, optionally followed by a sequence of \textit{subpunctis and prepunctis
+descriptor}s, optionally followed by a sequence of \textit{significant letter
+descriptor}s.
+
+The \textit{horizontal spacing adjustment descriptor}
+consists of a sequence of horizontal spacing adjustment characters
+\texttt{/} and \texttt{`}.
+\begin{itemize}
+  \item \texttt{//} move by \texttt{nabclargerspace} skip to the right
+  \item \texttt{/} move by \texttt{nabcinterelementspace} skip to the right
+  \item \texttt{``} move by \texttt{nabclargerspace} skip to the right
```

Comment:
I see from the example below that the backquote is a skip to the left.  You should probably correct that here and the next line.